.husky/pre-commit

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
```

Comment:
Please restore the `.husky/pre-commit` and `.husky/.gitignore` files as they are part of the lint process on commit.

Reply:
Done.